From dd42965048e5db15348644379a071d7667d71f50 Mon Sep 17 00:00:00 2001
From: Lars Wendler <polynomial-c@gentoo.org>
Date: Sat, 4 Jun 2016 19:59:21 +0200
Subject: [PATCH 3/5] Fixed sudo config for cpufreqnextgovernor

which needs an argument to run properly but sudo doesn't allow submission of an
argument.

Signed-off-by: Lars Wendler <polynomial-c@gentoo.org>
---
 INSTALL  | 2 +-
 Makefile | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/INSTALL b/INSTALL
index 773349f..44c3915 100644
--- a/INSTALL
+++ b/INSTALL
@@ -31,7 +31,7 @@ Post-Install
    the path to cpupower on your system):
    <user> <hostname> = (root) NOPASSWD: /usr/bin/cpupower -c [0-9]* frequency-set -g userspace
    <user> <hostname> = (root) NOPASSWD: /usr/bin/cpupower -c [0-9]* frequency-set -f [0-9]*
-   <user> <hostname> = (root) NOPASSWD: /usr/sbin/cpufreqnextgovernor
+   <user> <hostname> = (root) NOPASSWD: /usr/sbin/cpufreqnextgovernor [0-9]*
    where <user> is the user that runs gkrellm
    and <hostname> is the name of the host to set the CPU frequency/governor.
    Alternatively you may type 'make install-sudo', which tries to do this
diff --git a/Makefile b/Makefile
index 05a98ed..6753a47 100644
--- a/Makefile
+++ b/Makefile
@@ -22,7 +22,7 @@ install: cpupower.so
 install-sudo:
 	echo "$(USER) ALL = (root) NOPASSWD: /usr/bin/cpupower -c [0-9]* frequency-set -g userspace" >> /etc/sudoers
 	echo "$(USER) ALL = (root) NOPASSWD: /usr/bin/cpupower -c [0-9]* frequency-set -f [0-9]*" >> /etc/sudoers
-	echo "$(USER) ALL = (root) NOPASSWD: /usr/sbin/cpufreqnextgovernor" >> /etc/sudoers
+	echo "$(USER) ALL = (root) NOPASSWD: /usr/sbin/cpufreqnextgovernor [0-9]*" >> /etc/sudoers
 
 clean:
 	rm -f *.o core *.so* *.bak *~
-- 
2.8.3

